dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -153,15 +153,18 @@ func buildModules(srcPath, name, ufsType, moduleFlag, version string, modules ma
 		for _, arg := range strings.Split(moduleEntry.mavenArgs, " ") {
 			moduleMvnArgs = append(moduleMvnArgs, arg)
 		}
-		var versionMvnArg = "3.3.0"
-		for _, arg := range moduleMvnArgs {
-			if strings.Contains(arg, "ufs.hadoop.version") {
-				versionMvnArg = strings.Split(arg, "=")[1]
-			}
-		}
 		run(fmt.Sprintf("compiling %v module %v", name, moduleName), "mvn", moduleMvnArgs...)
+		if moduleEntry.ufsType != "" {
```

Comment:
the `ufsType` parameter is `"hdfs"` everywhere i see. we can be more direct; instead of passing it in as an argument, always use the one provided by `moduleEntry.ufsType`

Reply:
done